site/app/libraries/Output.php

```diff
@@ -78,6 +78,12 @@ public function loadTwig($full_load = true) {
             'cache' => $debug ? false : $cache_path,
             'debug' => $debug
         ]);
+
+        if($this->core->getConfig()->isDebug()){
```

Comment:
Actually, now that I'm looking at this again, change this to `if ($debug)` to use the variable defined above.